_posts/2018-08-22-cockpit-176.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cockpit 176
+author: Gundersanne
+date: 2018-08-22
+tags: cockpit linux
+slug: cockpit-176
+category: release
+summary: suggest browsers when browser is unsupported, auto-update backend install on demand
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 176.
+
+### Login: Suggest other browser when browser is unsupported
```

Comment:
This should have some explanation when this happens ("really old browsers that don't support WebSockets" and so on), https://github.com/cockpit-project/cockpit/issues/9438 might have some inspiration. It would not yet be shown for IE, but as that's slated for removal, I think this can just enumerate the known-good browsers (including Edge, not mentioning IE).